backend/server.js

```diff
@@ -36,76 +37,169 @@ app.use(
       secure: false,
       expires: new Date(Date.now() + 30 * 24 * 60 * 60 * 1000),
     },
-  }),
+  })
 );
 
 app.use(userRoutes);
 
-app.post("/player", async (req, res) => {
-  const {
-    AST,
-    BLK,
-    DRB,
-    ORB,
-    PF,
-    PTS,
-    STL,
-    TOV,
-    TRB,
-    age,
-    effect_fg_percent,
-    field_attempts,
-    field_goals,
-    field_percent,
-    ft,
-    ft_percent,
-    fta,
-    games,
-    games_started,
-    id,
-    minutes_played,
-    player_name,
-    team,
-    three_attempts,
-    three_fg,
-    three_percent,
-    two_attempts,
-    two_fg,
-    two_percent,
-  } = req.body;
-
-  const newPlayer = await prisma.player.create({
-    data: {
-      AST,
-      BLK,
-      DRB,
-      ORB,
-      PF,
-      PTS,
-      STL,
-      TOV,
-      TRB,
-      age,
-      effect_fg_percent,
-      field_attempts,
-      field_goals,
-      field_percent,
-      ft,
-      ft_percent,
-      fta,
-      games,
-      games_started,
+//GETS
+app.get("/players", async (req, res) => {
+  const { team, orderParam, ascending } = req.query;
```

Comment:
Gets all players in the database
<img width="1260" alt="Screenshot 2024-07-02 at 7 42 12 PM" src="https://github.com/GloriaXM/numbers-game/assets/112780860/e90f5b9d-f58a-404d-a82f-9b2a26b3a1a3">